src/azul/service/hca_response_v5.py

```diff
@@ -298,9 +308,53 @@ def make_projects(self, entry):
                 translated_project['insdcProjectAccessions'] = project.get('insdc_project_accessions', [None])
                 translated_project['insdcStudyAccessions'] = project.get('insdc_study_accessions', [None])
                 translated_project['supplementaryLinks'] = project.get('supplementary_links', [None])
+                translated_project['contributorMatrices'] = self.make_contributor_matrices_tree(entry)
             projects.append(translated_project)
         return projects
 
+    def make_contributor_matrices_tree(self, entry) -> JSON:
+        """
+        Returns a stratification tree for the Contributor-generated matrix files
+        in the entry.
+        """
+        contributor_matrices = entry['contents']['contributor_matrices']
+        if contributor_matrices:
+            files = one(contributor_matrices)['file']
+        else:
+            return {}
+        unique_values = defaultdict(set)  # Unique values per stratification key
+        for file in files:
+            # Transform the stratification string into a list of dicts (1 per line)
+            file['stratification'] = [
```

Comment:
This modifies the argument in place, which may be ok, but a version that does not is equally easy to understand.

Reply:
I recall asking you explicitly in a conversation to name this strata and stratum.